images/__common.sh

```diff
@@ -1,5 +1,6 @@
 __CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 source $__CURRENT_DIR/__sdkStorageConstants.sh
+source $__CURRENT_DIR/build/__common.sh
```

Comment:
we already have a common.sh, do not add another common.sh. Add your function to the existing common.sh